### DISCUSSION TITLE
Model Dutch RMBS

### DISCUSSION THREAD

**Discussion by yellowbean** [yellowbean/AbsBox#5] [Deal Modeling in AbsBox]
Missing part:
* swap structure
* revolving pools
* loan by loan headers
it seems that Dutch RMBS loan by loan data was disclosed at  [Loan by Loan](https://www.loanbyloan.eu/home/). but I don't have access to. Anyone who can access that ,it would be great to have the header/fields list available

Reference:
- [Loan by Loan](https://www.loanbyloan.eu/home/)
- [20190301 Dutch RMBS a primer.pdf](https://github.com/yellowbean/AbsBox/files/11025305/20190301.Dutch.RMBS.a.primer.pdf)
- [The+Danish+Mortgage+Bond+Market_UK.pdf](https://github.com/yellowbean/AbsBox/files/11329481/The%2BDanish%2BMortgage%2BBond%2BMarket_UK.pdf)
- [den-klassiske-realkreditmodel_uk_2021_final.pdf](https://github.com/yellowbean/AbsBox/files/11329482/den-klassiske-realkreditmodel_uk_2021_final.pdf)

https://github.com/yellowbean/AbsBox/issues/32

**Comment by yellowbean** [comment]
Let's use this deal as sample
[Hypenn VII - prospectus 11 June 2020.pdf](https://github.com/yellowbean/AbsBox/files/11851594/Hypenn.VII.-.prospectus.11.June.2020.pdf)
[Dutch Property Finance 2023-1 B.V..pdf](https://github.com/yellowbean/AbsBox/files/11862775/Dutch.Property.Finance.2023-1.B.V.pdf)

**Comment by yellowbean** [comment]
> loan by loan headers

Nope , EuroABS applies a strict rule which only allow stake holders to view the loan level data

**Comment by yellowbean** [comment]
PEARL Mortgage Backed Securities 3 B.V.  shall be a good start
[Fitch-report-Pearl-3.pdf](https://github.com/yellowbean/AbsBox/files/12308111/Fitch-report-Pearl-3.pdf)

**Comment by lucaborella89** [comment]
Hi @yellowbean we have started publishing data templates and ETL pipelines relevant for some dutch RMBS https://github.com/Algoritmica-ai/deeploans/tree/main/ETL-Pipelines/ECB-Loan-level-data-templates%20/Residential-Mortgages-ETL-Pipeline

**Reply by yellowbean** [reply]
Thanks @lucaborella89  thanks very helpful . While could you advise whether the header file (RMBS Taxonomy v28) is public ?  it it is accessible to anyone I'm happy to build helper function to convert loan tape into `absbox` friendly loan level input

**Reply by lucaborella89** [reply]
The RMBS Taxonomy V28 is public (see [here](https://www.ecb.europa.eu/mopo/coll/loanlevel/shared/files/RMBS_Loan_level_template.zip?84697a133af228836bd84d1e9dbbf9b5&84697a133af228836bd84d1e9dbbf9b5)) but not current anymore. We are going to work on the new template/header here https://github.com/Algoritmica-ai/deeploans/issues/1 @samgriek fyi

**Comment by yellowbean** [comment]
Close as complete at : https://absbox-doc.readthedocs.io/en/latest/nbsample/EDML_BLUE_2024.html
 
Engine changes:
* refactor on swap accrual and reference balance
* refactor on PDL
* new subordination interest via adding a new bond type which has two interest index
